content/pages/va-gov-onboarding-modal-check.html

```diff
@@ -0,0 +1,10 @@
+---
+production: false
+---
+<!doctype html>
+<html>
+  <head>
+      <meta http-equiv=refresh content="1; url=http://staging.va.gov">
+      <link rel=canonical href="/">
+  </head>
+</html>
```

Comment:
Because this modal is powered by `document.referrer`, I think we should have some sort of test for that modal appearing when redirected from Vets.gov to VA.gov. This will allow us to confirm that the onboarding modal appears when we navigate to `https://staging.vets.gov` and are redirected to `https://staging.va.gov`.